src/components/Card.js

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
-import '../styles/card.scss';
+import cx from 'classnames'
```

Comment:
Use `classnames` instead. There website is actually use both `cx` and `classnames` but `cx` is very misleading IMO.